aemedge/blocks/columns/columns.js

```diff
@@ -42,4 +42,32 @@ export default async function decorate(block) {
     const main = document.querySelector('main');
     await loadBlocks(main);
   }
+  const links = [
+    'https://main--sling--aemsites.aem.live/international/desi-tv/hindi',
+    'https://main--sling--aemsites.aem.live/international/desi-tv/dakshin',
+    'https://main--sling--aemsites.aem.live/international/desi-tv/hindi',
+    'https://main--sling--aemsites.aem.live/international/desi-tv/hindi',
+    'https://main--sling--aemsites.aem.live/international/desi-tv/hindi',
+    'https://main--sling--aemsites.aem.live/international/desi-tv/hindi',
+    'https://main--sling--aemsites.aem.live/international/desi-tv/hindi',
+    'https://main--sling--aemsites.aem.live/international/desi-tv/hindi',
+    'https://main--sling--aemsites.aem.live/international/desi-tv/dakshin',
+    'https://main--sling--aemsites.aem.live/programming/sports/soccer',
+    'https://main--sling--aemsites.aem.live/international/arabic',
+    'https://main--sling--aemsites.aem.live/international/eu/french',
+    'https://main--sling--aemsites.aem.live/international/eu/greek',
+    'https://main--sling--aemsites.aem.live/international/eu/italian',
+    'https://main--sling--aemsites.aem.live/international/eu/polish',
+    'https://main--sling--aemsites.aem.live/international/brazilian',
+    'https://main--sling--aemsites.aem.live/international/eastasian/cantonese',
+    'https://main--sling--aemsites.aem.live/international/eastasian/mandarin',
+    'https://main--sling--aemsites.aem.live/international/eastasian/taiwanese',
+  ];
+  const images = document.querySelectorAll('.columns div .columns-img-col img');
+  images.forEach((img, index) => {
+    const link = document.createElement('a');
+    link.href = links[index] || '#';
+    img.parentNode.insertBefore(link, img);
+    link.appendChild(img);
+  });
```

Comment:
Better would be to give the power to the authors for linking images anywhere they need to. If you put the link URL under each picture in your columns, then something like the below should do it (put this in "scripts"?) 
 
 ```
function decorateLinkedImages(main) {
  const pictures = main.querySelectorAll('picture');
  pictures.forEach((picture) => {
    const next = picture.nextElementSibling;
    if (next && next.tagName === 'A') {
      const a = next;
      a.replaceChildren(picture);
    } else if (next && next.tagName === 'BR' && next.nextElementSibling && next.nextElementSibling.tagName === 'A') {
      const a = next.nextElementSibling;
      a.replaceChildren(picture);
    }
  });
}
```